test/integration/preview_test.rb

```diff
@@ -0,0 +1,34 @@
+require 'test_helper'
+
+class PreviewTest < Capybara::Rails::TestCase
+
+
+	scenario "notifications have slide preview for sign owners" do 
+		# users ross and james are the only users that actually successfully log in
+		# Arrange
+		@sign = signs(:default)
```

Comment:
So im gonna be a @tatwell....it looks like the formatting is incorrect here, and you are maybe using 4 spaces instead of 2.